cassiopeia/preprocess/pipeline.py

```diff
@@ -577,22 +645,99 @@ def call_alleles(
     return alignments
 
 
+def error_correct_intbcs_to_whitelist(
+    input_df: pd.DataFrame,
+    whitelist_fp: Optional[str] = None,
+    whitelist: Optional[List[str]] = None,
```

Comment:
Same as above - let's group these two parameters together into a single `whitelist` parameter, and add a test to make sure this is supported.